daemon/main.go

```diff
@@ -735,7 +736,7 @@ func runDaemon() {
 	}
 
 	log.Info("Starting connection tracking garbage collector")
-	endpointmanager.EnableConntrackGC(!option.Config.IPv4Disabled, true)
+	endpointmanager.EnableConntrackGC(!option.Config.IPv4Disabled, true, viper.GetInt("conntrack-garbage-collector-interval"))
```

Comment:
Since this is a user input, we should do some more testing too, like a negative value?

Reply:
Thanks for reviewing, updated to use flags.UintP